docs/en/reference/batch-processing.rst

```diff
@@ -16,6 +16,15 @@ especially what the strategies presented here provide help with.
     operations.
 
 
+.. note::
+
+    Logger can have a serious memory and time impact on bulk treatments. 
+    Deactivating it can be a solution.
+.. code-block:: php
+
+    <?php
+    $em->getConnection()->getConfiguration()->setSQLLogger(null);
```

Comment:
The SQL logger isn't enabled by default in the ORM, so it might be worth mentioning that too. My suggestion would be something like this:

```
Having an SQL logger enabled when processing batches can have a serious impact on performance and resource usage.

To avoid that you should disable it in the DBAL configuration:

.. code-block:: php
    <?php
    $em->getConnection()->getConfiguration()->setSQLLogger(null);
```

Reply:
I think it's perfect ;-)